core/symbol/match_spec.rb

```diff
@@ -19,6 +19,8 @@
   it_behaves_like :symbol_match, :=~
 end
 
-describe "Symbol#match" do
-  it_behaves_like :symbol_match, :match
+ruby_version_is ""..."2.4" do
+  describe "Symbol#match" do
+    it_behaves_like :symbol_match, :match
+  end
```

Comment:
Could you add a case for the behavior in 2.4? Otherwise `Symbol#match` just becomes untested in >= 2.4.